File: DOTNET BACKEND/TransitHive/Models/BookingItem.cs
namespace TransitHive.Models
{
    public class BookingItem
    {
        public int Id { get; set; }
        public int BookingId { get; set; }
        public string Name { get; set; }

        public int Quantity { get; set; }

        public decimal BasePrice { get; set; }
        public Booking Booking { get; set; }
        
    }
}

File: SpringBootBackend/src/main/java/com/th/entity/BookingItem.java
package com.th.entity;

import java.math.BigDecimal;

import jakarta.persistence.Entity;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class BookingItem extends BaseEntity {

    @ManyToOne
    @JoinColumn(name = "booking_id")
    private Booking booking;

    private String name;
    private int quantity;
    private BigDecimal basePrice;
}
